docs/topics/lombok.md

```diff
@@ -0,0 +1,149 @@
+[//]: # (title: Lombok compiler plugin)
+
+> The Lombok compiler plugin is [Experimental](components-stability.md). 
+> It may be dropped or changed at any time. Use it only for evaluation purposes. 
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-7112).
+>
+{type="warning"}
+
+Since 1.5.20, Kotlin supports the experimental Lombok Kotlin compiler plugin. This plugin allows the use of
+Java's Lombok-generated declarations in mixed Kotlin/Java sources.
+If Java's Lombok-generated declarations are in another module than the Kotlin code that uses these declarations, 
+then you don't need to use this plugin.
+
+[Lombok](https://projectlombok.org/) does Java code generation itself, not the Lombok Kotlin compiler plugin. 
+So, you still need to configure Lombok as usual, when using this plugin. 
+Learn more about [how to make the plugin seeing Lombok's config](#defining-the-place-of-lombok-config).
+
+## Support of annotations
+
+Learn which annotations the plugin supports and the
+[current state of Lombok Kotlin compiler plugin](https://github.com/JetBrains/kotlin/blob/master/plugins/lombok/lombok-compiler-plugin/README.md).
+
+> Generation of `@Builder` annotation won't be supported in the current prototype. However, you can give your feedback
+> about the plugin and vote for [`@Builder` in YouTrack](https://youtrack.jetbrains.com/issue/KT-46959).
+>
+{type="warning"}
+
+> Kotlin compiler ignores Lombok annotations if you use them in Kotlin code.
+>
+{type="note"}
+
+Please read below about how to apply the Lombok Kotlin compiler plugin to your Gradle/Maven build.
```

Comment:
This line seems redundant because it doesn't affect anything.

What if we just rename Gradle and Maven headings to "Using in/with Gradle", "Using in/with Maven"? Does it make sense?

Reply:
Please don't use slash as "or"

Reply:
> What if we just rename Gradle and Maven headings to "Using in/with Gradle", "Using in/with Maven"? Does it make sense?

Actually we have such headings ("Gradle" and "Maven")for all other compiler plugins. Not sure that it's better than your idea, but it's consistent now.

Reply:
About this phrase – I stole it from another plugin's page)
Ok, will delete it.

>Please don't use slash as "or"
Ok.

>Actually we have such headings ("Gradle" and "Maven")for all other compiler plugins. 
This is why I used "Gradle" instead of "Using in Gradle", though had the same thought. So, I'll leave these headings as they are now.